README.md

```diff
@@ -390,6 +390,9 @@ guaranteed that the `DELETE` will be processed first (and vice versa).
 Deleting a grouping key without metrics is a no-op and will not result
 in an error.
 
+The body of a POST or PUT request may be gzip-compressed. Add a header Content-Encoding: gzip to do so.
+`echo "some_metric 3.14" | gzip | curl -H 'Content-Encoding: gzip' --data-binary @- http://pushgateway.example.org:9091/metrics/job/some_job`
```

Comment:
Please add a blank line before that. Otherwise it will all be in the same paragraph as the above. And perhaps start with "Example:". Then you could even use triple-backticks like this:

Example:
```bash
echo "some_metric 3.14" | gzip | curl -H 'Content-Encoding: gzip' --data-binary @- http://pushgateway.example.org:9091/metrics/job/some_job
```